pages/learn/index.vue

```diff
@@ -1,6 +1,47 @@
 <template>
   <article class="learn-page">
+    <OneXPBanner
+      copy="🎉 Qiskit is getting a new learning experience on IBM Quantum!"
+      :cta="{
+        label: 'Check it out!',
+        segment: {
+          cta: 'go-to-1xp-learning',
+          location: '1xp-banner',
+        },
+        url: 'https://example.com',
+      }"
+    />
     <LearnLeadSpace />
+    <OneXPPageSection
+      :features="[
+        {
+          descriptionParagraphs: [
+            'We are building a new home for the Qiskit Textbook on IBM Quantum, and we will be migrating courses and tutorials in the coming months.',
+            'Just getting started with Qiskit? Start your learning journey on the new platform today!',
+          ],
+          cta: {
+            label: 'Check it out',
+            segment: {
+              cta: 'go-to-1xp-learning',
+              location: '1xp-page-section',
+            },
+            url: 'https://example.com',
+          },
+          secondaryCta: {
```

Comment:
it seems like the design didn't intend for the "Learn more" to have an icon... 

(not blocking for me - I realize we're using our cta components and refactoring/updating them might not be priority rn)

![Screenshot 2023-08-11 at 6 16 48 AM](https://github.com/Qiskit/qiskit.org/assets/6276074/b173c23f-ecb0-4e8b-8938-5959978bd4b9)

Reply:
As discussed offline, these small design details are been discussed with the design team. It is known that our codebase won't reflect exactly the Figma design due to some structural differences, but that's OK for now.

In this case, we have to use the CTAs that we have in our codebase, which include the icons.